content/en/docs/setup/independent/high-availability.md

```diff
@@ -10,513 +10,217 @@ content_template: templates/task
+ - Three machines that meet [kubeadm's minimum
+  requirements](/docs/setup/independent/install-kubeadm/#before-you-begin) for
+  the control plane nodes
+- Three machines that meet [kubeadm's minimum
+  requirements](/docs/setup/independent/install-kubeadm/#before-you-begin) for
+  the workers
+- Three machines [TODO: simple kubelet and kubeadm] for etcd members
+- Full network connectivity between all machines in the cluster (public or
+   private network is fine)
 
-### Create etcd CA certs
+{{% capture prerequisites %}}
+{{% /capture %}}
 
-1. Install `cfssl` and `cfssljson`:
+# Common Steps
 
-     ```bash
-     curl -o /usr/local/bin/cfssl https://pkg.cfssl.org/R1.2/cfssl_linux-amd64
-     curl -o /usr/local/bin/cfssljson https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64
-     chmod +x /usr/local/bin/cfssl*
-     ```
-
-1. SSH into `etcd0` and run the following:
-
-      ```bash
-     mkdir -p /etc/kubernetes/pki/etcd
-     cd /etc/kubernetes/pki/etcd
-
-     cat >ca-config.json <<EOF
-     {
-        "signing": {
-            "default": {
-                "expiry": "43800h"
-            },
-            "profiles": {
-                "server": {
-                    "expiry": "43800h",
-                    "usages": [
-                        "signing",
-                        "key encipherment",
-                        "server auth",
-                        "client auth"
-                    ]
-                },
-                "client": {
-                    "expiry": "43800h",
-                    "usages": [
-                        "signing",
-                        "key encipherment",
-                        "client auth"
-                    ]
-                },
-                "peer": {
-                    "expiry": "43800h",
-                    "usages": [
-                        "signing",
-                        "key encipherment",
-                        "server auth",
-                        "client auth"
-                    ]
-                }
-            }
-        }
-     }
-     EOF
-     ```
-     ```bash
-     cat >ca-csr.json <<EOF
-     {
-        "CN": "etcd",
-        "key": {
-            "algo": "rsa",
-            "size": 2048
-        }
-     }
-     EOF
-     ```
+## Planning
 
-    {{< note >}}
-    **Optional:** You can modify `ca-csr.json` to add a section for `names`.
-    See [the CFSSL wiki](https://github.com/cloudflare/cfssl/wiki/Creating-a-new-CSR) for an example.
-    {{< /note >}}
+1. Figure out your pod CIDR. Please read the [CNI network
+   section](/docs/setup/independent/create-cluster-kubeadm/#pod-network) of the
+   docs for more information. Some CNI providers do not require a value to be
+   set. This guide uses calico for no particular reason. Because of this choice
```

Comment:
I would phrase this in a more positive way: 

"Some CNI providers do not require a value to be  set. This guide uses calico for no particular reason."

To something like:

 In this in example we have chose to use Calico (please note that some cni providers do not require special pod CIDR).

Reply:
Feedback on the feedback, I would not state that you're using Calico for "no particular reason."

Reply:
@mistyhacks exactly. Which is I why I'm suggesting he rephrased it to something more affirmative.

Reply:
I know we don't want to "pick winners" or "take sides", but there are very valid reason why Calico is a great choice for most use cases.

Reply:
The whole paragraph/step doesn't hold together very well, and any explanations are extraneous to a task-baseed doc. Try something like:
"1.  Find your pod CIDR. For details, see [the CNI network documentation](/docs/setup/independent/create-cluster-kubeadm/#pod-network). This example uses Calico, so the pod CIDR is `192.168.0.0/16`."